docs/05-awesomerc.md.lua

```diff
@@ -29,6 +29,7 @@ Here are the modules that we import:
 ]]
 
 sections.DOC_ERROR_HANDLING = [[
+
```

Comment:
Is this newline required?

Reply:
Yes, it changes how the markdown is expanded.